lib/cisco_node_utils/bgp_af.rb

```diff
@@ -244,7 +243,11 @@ def default_advertise_l2vpn_evpn
 
     # dampen_igp_metric
     def dampen_igp_metric
-      config_get('bgp_af', 'dampen_igp_metric', @get_args)
+      match = config_get('bgp_af', 'dampen_igp_metric', @get_args)
```

Comment:
Seems like the same issue as the discussion above. The correct fix is probably to restore the 'auto_default: false' and handle the case where config_get returns nil.